Presto hive and parquet CI tests are consistently failing

https://github.com/prestodb/presto/pull/28193

```
[ERROR]   TestParquetReader>AbstractTestParquetReader.testStructOfTwoNestedArrays:887 » NoSuchMethod 'com.facebook.presto.hive.$internal.parquet.org.apache.thrift.TFieldIdEnum org.apache.parquet.format.LogicalType.getSetField()'
[ERROR]   TestParquetReader>AbstractTestParquetReader.testStructOfTwoNestedSingleLevelSchemaArrays:908 » NoSuchMethod 'com.facebook.presto.hive.$internal.parquet.org.apache.thrift.TFieldIdEnum org.apache.parquet.format.LogicalType.getSetField()'
[ERROR]   TestParquetReader>AbstractTestParquetReader.testTimestampMillisBackedByINT64:1025 » NoSuchMethod 'com.facebook.presto.hive.$internal.parquet.org.apache.thrift.TFieldIdEnum org.apache.parquet.format.LogicalType.getSetField()'
[INFO] 
```

--- a/presto-hive/pom.xml
+++ b/presto-hive/pom.xml
@@ -73,36 +73,6 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.parquet</groupId>
-            <artifactId>parquet-column</artifactId>
-            <version>${dep.parquet.version}</version>
-            <exclusions>
-                <!-- excluding dependency due to vulnerability -->
-                <exclusion>
-                    <groupId>org.apache.commons</groupId>
-                    <artifactId>commons-lang3</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.parquet</groupId>
-            <artifactId>parquet-common</artifactId>
-            <version>${dep.parquet.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.parquet</groupId>
-            <artifactId>parquet-format-structures</artifactId>
-            <version>${dep.parquet.version}</version>
-            <exclusions>
-                <!-- excluding dependency due to vulnerability -->
-                <exclusion>
-                    <groupId>org.apache.thrift</groupId>
-                    <artifactId>libthrift</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-
-        <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-expressions</artifactId>
         </dependency>

--- a/presto-hudi/pom.xml
+++ b/presto-hudi/pom.xml
@@ -104,23 +104,6 @@
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-parquet</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.parquet</groupId>
-            <artifactId>parquet-column</artifactId>
-            <version>${dep.parquet.version}</version>
-            <exclusions>
-                <!-- excluding dependency due to vulnerability -->
-                <exclusion>
-                    <groupId>org.apache.commons</groupId>
-                    <artifactId>commons-lang3</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.parquet</groupId>
-            <artifactId>parquet-common</artifactId>
-            <version>${dep.parquet.version}</version>
-        </dependency>
 
         <dependency>
             <groupId>com.facebook.presto</groupId>

